content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -1,131 +1,172 @@
+:menuselection:`Orders Follow-up --> Last Order Sync`.
+
+Be sure to pick a date that occurs prior to the last status change of the desired order to
+synchronize and save.
 
 .. tip::
-   To synchronize immediately the orders of your Amazon account switch to :ref:`developer mode
-   <developer-mode>`, head to your Amazon account and click on **SYNC ORDERS**. The same can be done
-   with pickings by clicking on **SYNC PICKINGS**.
+   To immediately synchronize the orders of an Amazon account, switch to :ref:`developer mode
+   <developer-mode>`, head to the Amazon account in Odoo, and click :guilabel:`Sync Orders`. The
+   same can be done with pickings by clicking :guilabel:`Sync Pickings`.
 
 Manage deliveries in FBM
 ========================
 
-When a **FBM** (Fulfilled by Merchant) order is synchronized in Odoo, a picking is created along
-with the sales order and the customer. You can either ship all the ordered products to your customer
-at once or ship products partially by using backorders.
+Whenever an FBM (Fulfilled by Merchant) order is synchronized in Odoo, a picking is instantly
+created in the *Inventory* app, along with a sales order and customer. Then, decide to either ship
+all the ordered products to the customer at once, or ship products partially using backorders.
 
-When a picking related to the order is confirmed, a notification is sent to Amazon who will, in
-turn, notify the customer that the order (or a part of it) is on its way.
+When a picking related to the order is confirmed, a notification is then sent to Amazon, who, in
+turn, notifies the customer that the order (or a part of it) is on its way.
 
 .. important::
-   Amazon requires to provide a tracking reference with each delivery. You'll need to assign a
-   carrier. If the carrier doesn't automatically provide a tracking reference, you'll need to set
-   one manually. This concerns all marketplaces.
+   Amazon requires users to provide a tracking reference with each delivery. This is needed to
+   assign a carrier.
+
+   If the carrier doesn't automatically provide a tracking reference, one must be set manually. This
+   rule applies to all marketplaces.
 
 .. tip::
-   - If your chosen carrier isn't one supported by Odoo, you can still create a carrier bearing its
-     name (e.g. create a carrier named `Colissimo`). This name is case insensitive, but be careful
-     about typos, as Amazon won't recognize them.
-   - Create a delivery carrier named `Self Delivery` to inform Amazon that you make your own
-     deliveries. You still have to enter a tracking reference.
-   - Keep in mind that the customer is notified by email about the delivery, and the carrier and
-     tracking reference are displayed in the email to the customer.
+   - If the chosen carrier isn't supported by Odoo, a carrier with the same name can still be
+     created (e.g. create a carrier named `Colissimo`). The name used is **not** case sensitive, but
```

Comment:
This example is too localized and would confuse anyone not in the EU.
Would change to `DHL` or `FedEx` maybe for more universal understanding.

Reply:
We support both DHL and Fedex and this may cause some folks to think otherwise. I'm going to say EasyShip as a neutral example of a carrier not supported by Odoo.